src/pages/index.tsx

```diff
@@ -29,17 +29,25 @@ export default function Index() {
     return (
       <div className="app">
         <h1 className={"title"}>Linear Style</h1>
-        <span className="subtitle">Fetching the latest themes</span>
+        <span className="subtitle">🧪 Hold up! Fetching the latest themes...</span>
       </div>
     );
 
   return (
     <div className="app">
       <h1 className="title">Linear Style</h1>
-      <span className="subtitle">Click on a theme to copy it.</span>
+      <span className="subtitle">Click on a theme to copy it. Then, paste into your Linear.</span>
```

Comment:
This subtitle was fine text wise as it was before, I think this is too _waffly_ 😝